common/app/views/fragments/items/facia_cards/contentCard.scala.html

```diff
@@ -88,10 +88,10 @@
                 </div>
             }
 
-            case Some(media@InlineYouTubeMediaAtom(_)) => {
+            case Some(media@InlineYouTubeMediaAtom(_)) if item.cardTypes.showYouTubeMediaAtomPlayer => {
```

Comment:
So if showYouTubeMediaAtomPlayer is false we don't render the youtube template fragment at all? I wonder if we could still use this template but pass showYouTubeMediaAtomPlayer as an argument. We can then put the logic to check this argument in that template; if true we add the iframe and if false we add the play button and overlay (which link through to the video page). +'ve is It'd save us having a new template? -'ve is more logic in that template.